packages/components/src/utils/getPropsFrom.js

```diff
@@ -15,10 +15,12 @@ function extractComponentProps(Component, props) {
 
 function extractNativeProps(props) {
 	const extractedProps = {};
-	NATIVE_PROPS.forEach(propName => {
-		if (props[propName] !== undefined) {
-			extractedProps[propName] = props[propName];
-		}
+	NATIVE_PROPS.forEach(nativeProp => {
+		Object.keys(props).forEach(propName => {
+			if (propName.includes(nativeProp)) {
```

Comment:
startswith looks safer to me.
or fill the NATIVE_PROPS with regex so you just do nativeProp.test(propName) ?

Reply:
I switched for `startsWith` :+1: